stdlib/contrib/sranka/sensu/sensu.flux

```diff
@@ -47,14 +131,70 @@ event = (
     return http.post(headers: headers, url: url + "/api/core/v2/namespaces/" + namespace + "/events", data: enc)
 }
 
-// `endpoint` creates a factory function that creates a target function for pipeline `|>` to send event to Sensu for each table row.
-// `url` - string - base URL of [Sensu API](https://docs.sensu.io/sensu-go/latest/migrate/#architecture) without a trailing slash, for example "http://localhost:8080" .
-// `apiKey` - string - Sensu [API Key](https://docs.sensu.io/sensu-go/latest/operations/control-access/).
-// `handlers` - array<string> - Sensu handlers to execute.
-// `namespace` - string - The Sensu namespace. Defaults to "default".
-// `entityName` - string - Source of the event, it can contain [a-zA-Z0-9_.\-] characters, other characters are replaced by underscore. Defaults to "influxdb".
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `checkName`, `text`, and `status`, as defined in the `event` function arguments.
+// endpoint sends an event
+// to the [Sensu Events API](https://docs.sensu.io/sensu-go/latest/api/events/#create-a-new-event)
+// using data from table rows.
+//
+// ## Parameters
+// - url: Base URL of [Sensu API](https://docs.sensu.io/sensu-go/latest/migrate/#architecture)
+//   *without a trailing slash*.
+//   Example: `http://localhost:8080`.
+// - apiKey: Sensu [API Key](https://docs.sensu.io/sensu-go/latest/operations/control-access/).
+// - handlers: [Sensu handlers](https://docs.sensu.io/sensu-go/latest/reference/handlers/) to execute.
+//   Default is `[]`.
+// - namespace: [Sensu namespace](https://docs.sensu.io/sensu-go/latest/reference/rbac/).
+//   Default is `default`.
+// - entityName: Event source.
+// - Default is `influxdb`.
+//
+//   Use alphanumeric characters, underscores (`_`), periods (`.`), and hyphens (`-`).
+//   All other characters are replaced with an underscore.
+//
+// - tables: Input data. Default is piped-forward data (`<-`).
```

Comment:
It looks like CI is also complaining about this. I think removing it should appease the parser, but I'm not sure.